infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -165,11 +166,12 @@ private void initStreamLogDirectory() {
                 throw new LogUnitException("Cannot start Corfu on a read-only filesystem:" + corfuDir);
             }
 
-            if (logSizeLimit != -1 && logSizeLimit > corfuDirBackend.getTotalSpace()) {
-                String msg = String.format("Invalid quota: quota(%d) is greater than the file store size (%d)",
-                        logSizeLimit, corfuDirBackend.getTotalSpace());
+            if (logSizeLimitPercentage < 0.0 || 100.0 < logSizeLimitPercentage) {
```

Comment:
Not sure whether we need to consider the precision of double comparison here. Generally speaking, when comparing double numbers, epsilon is usually used. 

https://stackoverflow.com/questions/356807/java-double-comparison-epsilon

Reply:
We could, but here it is a simple boundary sanity check right, so do we need precision at all?

Reply:
Yes. I share with your opinion. Sanity check does not need to be strict.